scipy/stats/contingency.py

```diff
@@ -271,3 +272,122 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+def association(arr, chi2_stat, stat="cramer"):
```

Comment:
Could you rename `arr` -> `observed` for consistency with the other functions in this namespace. And `stat` to `method` - more descriptive, and many other functions use this name.

Reply:
done.